src/Service/Models/PeriodicCredentialReport.cs

```diff
@@ -0,0 +1,19 @@
+namespace Passwordless.Service.Models;
+
+public class PeriodicCredentialReport : PerTenant
+{
+    public DateTime CreatedAt { get; set; }
```

Comment:
Should we perhaps tweak this property to always return midnight so that we never accidently include current time (which would result in double reporting)

Reply:
Or perhaps: https://learn.microsoft.com/en-us/dotnet/standard/datetime/how-to-use-dateonly-timeonly

Reply:
I think I'm already doing this a layer up or so.

Reply:
`public async Task<int> UpdatePeriodicCredentialReportsAsync()
    {
        var result = _db.AccountInfo
            .GroupJoin(
                _db.Credentials,
                accountInfo => accountInfo.Tenant,
                credential => credential.Tenant,
                (accountInformation, credentials) => new PeriodicCredentialReport
                {
                    Tenant = accountInformation.Tenant,
                    UsersCount = credentials.Select(x => x.UserId).Distinct().Count(),
                    CredentialsCount = credentials.Count(),
                    CreatedAt = _timeProvider.GetUtcNow().UtcDateTime.Date // here
                });

        await _db.PeriodicCredentialReports.AddRangeAsync(result);

        var rows = await _db.SaveChangesAsync();

        return rows;
    }`